src/main/java/com/esri/samples/ogc/browse_wfs_layers/BrowseWfsLayersSample.java

```diff
@@ -0,0 +1,229 @@
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.CheckBox;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import java.awt.*;
+import java.util.List;
+import java.util.Random;
+
+public class BrowseWfsLayersSample extends Application {
+
+  private MapView mapView;
+  private CheckBox checkBox;
+  private ArcGISMap map;
+  private ProgressIndicator progressIndicator;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    // create stack pane and JavaFX app scene
+    StackPane stackPane = new StackPane();
+    Scene scene = new Scene(stackPane);
+    scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+    // set title, size, and add JavaFX scene to stage
+    stage.setTitle("Browse WFS Layers");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+
+    // create a control panel
+    VBox controlsVBox = new VBox(6);
+    controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY, Insets.EMPTY)));
+    controlsVBox.setPadding(new Insets(10.0));
+    controlsVBox.setMaxSize(170, 220);
+    controlsVBox.getStyleClass().add("panel-region");
+
+    // create a button to load selected layer
+    Button loadButton = new Button("Load selected layer");
+    loadButton.setMaxWidth(Double.MAX_VALUE);
+
+    // create a check box to swap coordinate order
+    checkBox = new CheckBox("Swap coordinate order");
+    checkBox.setMaxWidth(Double.MAX_VALUE);
+
+    // create a list to hold the names of the bookmarks
+    ListView<WfsLayerInfo> wfsLayerNamesListView = new ListView<>();
+    wfsLayerNamesListView.setMaxHeight(160);
+
+    // create a progress indicator
+    progressIndicator = new ProgressIndicator();
+    progressIndicator.setVisible(false);
+
+    // create an ArcGISMap with topographic basemap and set it to the map view
+    map = new ArcGISMap(Basemap.createImagery());
+    mapView = new MapView();
+    mapView.setMap(map);
+
+    // URL to the WFS service
+    String serviceUrl = "https://dservices2.arcgis.com/ZQgQTuoyBrtmoGdP/arcgis/services/Seattle_Downtown_Features/WFSServer?service=wfs&request=getcapabilities";
+
+    // create a WFS service
+    WfsService wfsService = new WfsService(serviceUrl);
+    wfsService.loadAsync();
+    wfsService.addDoneLoadingListener(() -> {
```

Comment:
add some comment about browsing the layers in the service since that is the main point of the sample